document_url/__manifest__.py

```diff
@@ -7,6 +7,7 @@
     "author": "Tecnativa, Odoo Community Association (OCA)",
     "website": "https://github.com/OCA/knowledge",
     "license": "AGPL-3",
+    "depends": ["web"],
```

Comment:
Shouldn't it be mail?

Reply:
I do not see that it is necessary... but I am not sure..

Reply:
Seeing this:

https://github.com/OCA/knowledge/blob/f79f13724ff1aed24b83f23b2a79bf42ef22ae16/document_url/static/src/js/url.js#L11

I think it should be.